threescale_api/resources.py

```diff
@@ -795,6 +837,37 @@ def app_plan(self) -> ApplicationPlan:
         return self.parent
 
 
+class Backend(DefaultResource):
+    def __init__(self, entity_name='system_name', **kwargs):
+        super().__init__(entity_name=entity_name, **kwargs)
+
+    @property
+    def metrics(self) -> Metrics:
+        return BackendMetrics(parent=self, instance_klass=Metric)
```

Comment:
Done

Reply:
Maybe I was misunderstood? look at `instance_klass=Metric` shouldn't it be `instance_klass=BackendMetric` ? Or otherwise should the `BackendMetric` exist at all?

Reply:
@mkudlej ^^